### DISCUSSION TITLE
Have Flowise one click flowise installation

### DISCUSSION THREAD

**Discussion by acmhmedia** [coollabsio/coolify#2849] [Service Template Requests]
Be able to on-click install Flowise with coolify. Flowise is a no code open source LLM chain builder. I think it can be useful for many and espacially because it's a no code tool noobs like me want to be able to one click install it on my server with coolify.
Related links:
https://docs.flowiseai.com/getting-started
https://github.com/FlowiseAI/Flowise/tree/main

**Comment by taowang1993** [comment]
You can deploy flowise with "Existing Docker Image" using Coolify.
Copy this: flowiseai/flowise:latest
Remember to configure your enviroment variables.

**Comment by klugjo** [comment]
- Create project, add new resource and click on "Docker Image" (Existing docker image from any Registry, without Git)
- Paste `flowiseai/flowise:latest` as the docker image and click save
- Set the name of your configuration as well as the domain to whatever you want or leave the defaults to try things out
- Click Deploy on the top right and you are done
- Optionally, you can set the `FLOWISE_USERNAME` and `FLOWISE_PASSWORD` env variables to protect your flowise installation

**Comment by chrislsww** [comment]
Thanks. Is there any way to change the login screen? I'd love to show just the login fields, but not "Chatflows" and "Add new". Also, does Flowise support 2FA yet?
<img width="1161" alt="Bildschirmfoto 2024-11-02 um 19 19 09" src="https://github.com/user-attachments/assets/90f0388f-c540-467a-85ac-4d2ea4ddd582">

**Comment by DmacMcgreg** [comment]
I get the above login page with no way to actually login when deploying on coolify

**Comment by klugjo** [comment]
@DmacMcgreg did you add the env variables?

**Comment by DmacMcgreg** [comment]
I did yes.

**Comment by DmacMcgreg** [comment]
@klugjo Just tried again with coolify on a completely different server.

FLOWISE_USERNAME = user
FLOWISE_PASSWORD = 1234567

When I type them in, the modal disappears then reappears after 1 second
